### PR TITLE
doc: add make for Ubuntu, and make the CMake instructions a little more active

### PR DESCRIPTION
**Pull request by SvenDowideit**
wrt the CMake dependency, Ubuntu 17.10 has CMake 3.9.1.

### CODE REVIEW

File: doc/getting_started/installation_linux.rst [hunk 1]
```diff
@@ -69,8 +69,8 @@ Install additional packages required for development with Zephyr::
 
    $ pip3 install --user -r scripts/requirements.txt
 
-Note that CMake version 3.8.2+ or higher is required. If you are using a
-distribution that currently ships an older version, check the backports or
+CMake version 3.8.2+ or higher is required. Check what version you have using
+`cmake --version`, and if its an older version, check the backports or
```
Comment (pfalcon) [inline doc/getting_started/installation_linux.rst]: double backticks

File: doc/getting_started/installation_linux.rst [hunk 2]
```diff
@@ -69,8 +69,8 @@ Install additional packages required for development with Zephyr::
 
    $ pip3 install --user -r scripts/requirements.txt
 
-Note that CMake version 3.8.2+ or higher is required. If you are using a
-distribution that currently ships an older version, check the backports or
+CMake version 3.8.2+ or higher is required. Check what version you have using
```
Comment (pfalcon) [inline doc/getting_started/installation_linux.rst]: Please take chance to remove "+" from "3.8.2+". It should be either "version 3.8.2+" or "version 3.8.2 or higher", the latter will be understood by more people.

**Comment by SvenDowideit** [comment]
@pfalcon 👍

**Review by carlescufi — APPROVED** [review]
Nice additions

**Comment by nashif** [comment]
fix one of the commit messages:

1: UC5 Title exceeds max length (78>72): "doc: add make for Ubuntu, and make the CMake instructions a little more active"

**Pull request merged**